test_unstructured/metrics/test_element_type.py

```diff
@@ -35,3 +38,37 @@ def test_get_element_type_frequency(filename, frequency):
     elements = partition(filename=f"example-docs/{filename}")
     elements_freq = get_element_type_frequency(elements_to_json(elements))
     assert elements_freq == frequency
+
+
+@pytest.mark.parametrize(
+    ("filename", "frequency"),
+    [
+        (
+            "fake-email.txt",
+            {
+                ("UncategorizedText", None): 6,
+                ("ListItem", None): 12,
+                ("Title", None): 5,
+                ("NarrativeText", None): 2,
+            },
+        ),
+        (
+            "sample-presentation.pptx",
+            {
+                ("Title", 0): 4,
+                ("Title", 1): 1,
+                ("NarrativeText", 0): 3,
+                ("ListItem", 0): 6,
+                ("ListItem", 1): 6,
+                ("ListItem", 2): 3,
+                ("Table", None): 1,
+            },
+        ),
+    ],
+)
+def test_calculate_element_type_percent_match(filename, frequency):
+    elements = partition(filename=f"example-docs/{filename}")
+    elements_freq = get_element_type_frequency(elements_to_json(elements))
+    assert calculate_element_type_percent_match(elements_freq, frequency) == 1.0
```

Comment:
so in these tests, are the elements always 100% match? Can we get some examples (we can just modify the "ground truth" parametrized element frequencies) where we don't have 100%, either through missing or misclassified elements?

Reply:
We could mock fake output, but it'd be best if we compare the mock one against the gold standard. So if we want to see a comparison from same file but two sources, let's create a gold standard for these files.

Reply:
Ok sounds good. It can possibly just be the unstructured output with manual human edits to look like the gold standard, we don't need to make it from scratch.

I just want to see a case where the unstructured output somehow doesn't 100% match the gold standard, however that test can be added is up to you :)